examples/images/cifar10/train_cifar10.py

```diff
@@ -169,5 +175,17 @@ def train(argv):
                 )
 
 
+def main(argv):
+    # get world size (number of GPUs)
+    world_size = int(os.getenv("WORLD_SIZE", 1))
```

Comment:
I feel like world_size is not really a good name. What do you think of calling it 'total_number_of_gpus' or something in that spirit?

Reply:
I think it's quite common to call this variable `world_size`, but I admit that `total_num_gpus` or sth alike is more descriptive, so I changed the variable name accordingly